packages/enzyme-adapter-react-13/src/ReactThirteenAdapter.js

```diff
@@ -129,7 +129,7 @@ class ReactThirteenAdapter extends EnzymeAdapter {
             Component: type,
             props,
             context,
-            ...(ref && { ref }),
+            ...(ref && { forwardedRef: ref }),
```

Comment:
what happens when this new version of the 13 adapter interacts with an older version of `enzyme`, that lacks support for the `forwardedRef` prop?

Reply:
@ljharb  Good question. We could use some marker on either `createMountWrapper` or its return value to indicate if a `forwardedRef` prop is supported. `ReactWrapperComponent.propTypes.forwardedRef !== undefined` might be sufficient already but this coupling to propTypes is pretty dangerous since you would expect that you can safely remove it. I would probably just add a static `supportsForwardedRefProp` or `supportedProps` field to the create component in `createMountWrapper` and check that. Sounds good?

Reply:
That seems reasonable. The important thing is that "old adapter + new enzyme" and "new adapter + old enzyme" do not break in any way - it's totally fine if a bug is only fixed with "new adapter + new enzyme".